server/index.js

```diff
@@ -67,11 +72,24 @@ require('./global/api/router')(router, app)
 
 app.use('/', router);
 
+// serve the react app index.html
+app.get('*', (req, res) => {
```

Comment:
can we move this back into the router.js? is this the path issue you were talking about?

Reply:
I can put it back there with no issues. The only reason I moved it is because index.js already uses `buildPath`. I'll put it back.